carbon-design-system/carbon
# separate categories.yml from metadata.yml

Closes #3499 

**changed**
- removes categories info from `metadata.yml` and puts it in `categories.yml`
- updates `build-metadata.js` to use this new file

**testing**
- run `node tasks/build-metadata.js` 
- verify that `the output file `icons/metadata.json` contains this info at the bottom as expected (should be unchanged)
